src/common/libsubprocess/subprocess.h

```diff
@@ -103,14 +103,14 @@ typedef struct {
 /*
  *  llog-compatible callback
  */
-typedef void (*subprocess_log_f) (void *arg,
```

Comment:
I think I noticed `command.h` has a similar issue.

Reply:
Ah good catch.  I went ahead and added a vi mode line to all headers in libsubprocess and fixed command.h.